.github/CONTRIBUTING.md

```diff
@@ -13,6 +18,7 @@ submit a pull request. Please, try to follow these guidelines when you do so.
 * Open an issue with a descriptive title and a summary in grammatically correct, complete sentences.

```

Comment:
I am for my own writing for sure. :-) But I think it hampers contributions. I'd rather have a badly spelled great idea than none.